data/verb-forms.json

```diff
@@ -254,6 +254,265 @@
                     "ru": "делают"
                 }
             }
-        ]
+        ],
+        "type": "sahih"
+    },
+    {
+        "base": "قَالَ",
+        "forms": [
+            {
+                "tense": "past",
+                "pronoun": "fsb",
+                "text": {
+                    "ar": "قُلْتُ",
+                    "en": "said",
+                    "ru": "сказал"
+                }
+            },
+            {
+                "tense": "past",
+                "pronoun": "fpb",
+                "text": {
+                    "ar": "قُلْنَا",
```

Comment:
I was omitting fatha if there is alif after it.

Reply:
Let's see it as a minor thing for now.